docs_2.0/manual_EN/5.Deployment and Installation/1.resource_preparations.md

```diff
@@ -0,0 +1,108 @@
+
+## Requirements for compiling Nebula Graph
+
+TODO
+
+## Requirements and suggestions for installing Nebula Graph in test environments
+
+### Hardware requirements for test environments
+
+| Item               | Requirement |
+| ------------------ | ----------- |
+| CPU architecture   | x86_64      |
+| Number of CPU core | 4           |
+| Memory             | 8 GB        |
+| Storage            | 100 GB, SSD |
+
+### Supported operation systems for test environments
+
+For now, we can only install Nebula Graph in the Linux system.
+
+To install Nebula Graph in a test environment, we recommend that you use any Linux system with kernel version 2.6.32 or higher.
+
+### Suggested service architecture for test environments
+
+* 3 metadata service processes (metad)
+* At least 3 storage service processes (storaged)
+* At least 3 query engine service processes (graphd)
+
+Each metad process automatically creates and maintains a copy of the metadata. Usually, you only 3 metad processes. The number of storaged processes does not affect the number of graph space copies.
+
+You can deploy multiple processes on a single machine. For example, on a cluster of 5 machines (named as A, B, C, D and E), you can deploy Nebula Graph as follows:
+
+> **WARNING**: Do not deploy a cluster across IDCs.
+
+* A: 1 metad, 1 storaged, 1 graphd
+* B: 1 metad, 1 storaged, 1 graphd
+* C: 1 metad, 1 storaged, 1 graphd
+* D: 1 storaged, 1 graphd
+* E: 1 storaged, 1 graphd
+
+### Software requirements for test environments
+
+See [General software requirements](#general_software_requirements).
+
+## Requirements and suggestions for installing Nebula Graph in production environments
+
+### Hardware requirements for production environments
+
+| Item               | Requirement          |
+| ------------------ | -------------------- |
+| CPU architecture   | x86_64               |
+| Number of CPU core | 48                   |
+| Memory             | 96 GB                |
+| Storage            | 2 * 900 GB, NVMe SSD |
+
+### Supported operation systems for production environments
+
+For now, we can only install Nebula Graph in the Linux system.
+
+To install Nebula Graph in a production environment, we recommend that you use any Linux system with kernel version 3.9 or higher.
+
+### Suggested service architecture for production environments
+
+* 1 metadata service process (metad)
+* At least 1 storage service process (storaged)
+* At least 1 query engine service process (graphd)
+
+For example, on a cluster of 3 machines (named as A, B, and C), you can deploy Nebula Graph as follows:
+
+* A: 1 metad, 1 storaged, 1 graphd
+* B: 1 storaged, 1 graphd
+* C: 1 storaged, 1 graphd
+
+### Software requirements for production environments
+
+See [General software requirements](#general_software_requirements).
+
+## General software requirements
```

Comment:
This is for compiling, not for installing.

Reply:
Thank you. I moved this part to the Requirements for compiling Nebula Graph section and removed its links in other sections.